src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
-'use strict';
+"use strict";
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
```

Comment:
If you follow the suggestion above;

```suggestion
  constructor(name) {
    super(name);
```
class extending would just need the name.